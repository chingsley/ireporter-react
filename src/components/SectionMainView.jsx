import React from 'react';

const mainSection = () => (
  <section className="main-section">
    <h1 className="mission-statement-big">
      <span className="mission-statement-big__style-ireporter">
        <span className="mission-statement-big__style-ireporter--style-i">i</span>
        <span className="mission-statement-big__style-ireporter--style-reporter">Reporter</span>
      </span>
      <span className="mission-statement-big__style-africa">Africa</span>
    </h1>
    <h3 className="mission-statement-small">
      breaking barriers and creating a better future
    </h3>
  </section>
);

export default mainSection;
