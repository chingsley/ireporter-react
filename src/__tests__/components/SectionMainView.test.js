import { shallow } from 'enzyme';
import React from 'react';
import SectionMainView from '../../components/SectionMainView';
// import { shallowWrap } from '../helpers/componentWrapper';

describe('SectionMainView component on the home page', () => {
  it('should render homepage correctly', () => {
    const wrapper = shallow(<SectionMainView />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should contain a section with class .main-section', () => {
    const wrapper = shallow(<SectionMainView />);
    expect(wrapper.find('.main-section').exists()).toBe(true);
  });
});
