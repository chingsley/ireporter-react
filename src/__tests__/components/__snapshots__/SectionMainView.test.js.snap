// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionMainView component on the home page should render homepage correctly 1`] = `ShallowWrapper {}`;
